src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -59,6 +59,11 @@
         action = component.get("c.getAddresses");
         action.setParams({ hhId : hhId, listAddrExisting : null });
         action.setCallback(this, function(response) {
+
+		    // tell our visualforce page we are done loading
```

Comment:
Picked one, liked the one that allowed maximum control. :)  But, for those with the left-pinky twitch <ahem>:

https://atom.io/packages/tabs-to-spaces
http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs
https://www.sublimetext.com/docs/2/indentation.html